vue.config.js

```diff
@@ -33,6 +33,11 @@ module.exports = {
       config.resolve.alias.set('vue$', 'vue/dist/vue.esm-bundler.js');
     }
   },
+  css: {
+    loaderOptions: {
+      css: { url: false }
+    }
+  },
```

Comment:
The build didn't like icomoon.css: something about the asset URLs. I made this change based on vuejs/vue-cli#7032, but I'll take another look at this later.